packages/cli/src/lib/implementation/collect-and-persist.ts

```diff
@@ -29,7 +28,7 @@ export async function collectAndPersistReports(
       // Running checks after persisting helps while debugging as you can check the invalid output after the error
       pluginOutputSchema.parse(plugin);
     } catch (e) {
-      throw new CollectOutputError(plugin.slug, e as Error);
+      throw new Error(plugin.slug + (e as Error).message);
```

Comment:
There's no space between slug and message, so it wouldn't be very readable. I recommend using template strings instead of concatenation, it's easier to spot such issues this way:

```suggestion
      throw new Error(`${plugin.slug} - ${(e as Error).message}`);
```